atlp-rwanda/eagles-ec-be
# #187419168 Logout Functionality

#### What does this PR do?
This PR implements the logout functionality

#### Description of Task to be completed?
- add logout functionality
- add feature documentation
- test the feature

#### How should this be manually tested?
1. Clone the repository.
2. Checkout to the branch `ft-logout-#187419168`.
3. Run `npm install` to install dependencies.
5. Run `npm run dev` to start the development server.
6. Open a web browser and navigate to the `/docs` endpoint (e.g., `http://localhost:PORT/docs`).
7. Verify that the Swagger UI loads correctly.
8. First authorize by logging in any user and use the token provided
9. Explore the endpoint `/api/v1/users/logout` for logging out and verify that everything is working.

#### What are the relevant pivotal tracker stories?

[Delivers #187419168]
